README.rst

```diff
@@ -1,4 +1,5 @@
 來自 https://speakerdeck.com/mosky/get-power-from-command-line \
 的寫作計畫。
 
+文章網址: `cmdpower.readthedocs.io <https://cmdpower.readthedocs.io/zh_TW/latest/>`_
```

Comment:
I missed one thing: You need a blank line between this and the next line.  Could you please fix it (using another PR)?